blazor/appearance/themes.md

```diff
@@ -977,6 +977,29 @@ Material and Tailwind Themes uses online roboto font. If your app is designed to
         <link href="material.min.css" rel="stylesheet" />
     </head>
     ```
+### Using Customized Styles from Syncfusion.Blazor.Themes Package and CDN (Version 23.2.4 and above) 
+
+Starting from version 23.2.4, customized Material and Tailwind themes are available in the Syncfusion.Blazor.Themes package and can be accessed via CDN for Blazor components.
```

Comment:
Starting from v23.2.4

Reply:
The changes have been made as per the suggestion.